htsohm/db/__init__.py

```diff
@@ -23,6 +23,9 @@
 from htsohm.db.surface_area import SurfaceArea
 from htsohm.db.void_fraction import VoidFraction
 from htsohm.db.material import Material
+from htsohm.db.structure import Structure
```

Comment:
Presumably you grabbed all this code from a specific commit–would be nice to know what commit sha this came from for reference.